scripts/tasks/dev.ts

```diff
@@ -3,26 +3,32 @@ import detectFreePort from 'detect-port';
 import type { Task } from '../task';
 import { exec } from '../utils/exec';
 
-export const START_PORT = 6006;
+export const PORT = process.env.STORYBOOK_SERVE_PORT
+  ? parseInt(process.env.STORYBOOK_SERVE_PORT, 10)
+  : 6006;
 
 export const dev: Task = {
   description: 'Run the sandbox in development mode',
   service: true,
   dependsOn: ['sandbox'],
   async ready() {
-    return (await detectFreePort(START_PORT)) !== START_PORT;
+    return (await detectFreePort(PORT)) !== PORT;
   },
   async run({ sandboxDir, codeDir }, { dryRun, debug }) {
     const controller = new AbortController();
+    const devCommand = `yarn storybook --port ${PORT}${
+      process.env.SELECTED_TASK === 'dev' ? '' : ' --ci'
+    }`;
```

Comment:
Maybe we could just add a flag for this to the task runner? IDK.

Reply:
I thought of that, but didn't know if it was the right approach as it didn't seem to be related to TemplateDetails

Reply:
Done!